source/operations/concepts.rst

```diff
@@ -8,126 +8,163 @@ Core Operational Concepts
+-----------------------------------------------------------------
+
+Store objects resiliently with :ref:`Erasure Coding <minio-erasure-coding>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MinIO Erasure Coding is a data redundancy and availability feature that allows MinIO deployments with multiple drives to automatically reconstruct objects on-the-fly despite the loss of multiple drives or nodes in the cluster. 
+Erasure Coding provides object-level healing with significantly less overhead than adjacent technologies such as RAID or replication.
+
+
+Identify and correct hidden errors caused by bit rot
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Bit rot is the random, silent corruption to data that can happen on any storage device.
+Bit rot corruption is not prompted by any activity from a user, nor does the system's operating system alone have awareness of the corruption to notify a user or administrator about a change to the data.
+
+Some common reasons for bit rot include:
+     
+- ageing drives
+- current spikes
+- bugs in disk firmware
+- phantom writes
+- misdirected reads/writes
+- driver errors
+- accidental overwrites
+
+MinIO uses a hashing algorithm to confirm the integrity of an object.
+If an object becomes corrupted by bit rot, MinIO can automatically heals the object depending on the system topology and availability.
+
+Divide space responsibly with :ref:`erasure sets <minio-ec-erasure-set>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An erasure set is a group of multiple drives that supports MinIO :ref:`Erasure Coding <minio-erasure-coding>`. 
+Erasure Coding provides high availability, reliability, and redundancy of data stored on a MinIO deployment.
+
+MinIO divides objects into chunks — called `shards` — and evenly distributes them among each drive in the Erasure Set. 
+MinIO can continue seamlessly serving read and write requests despite the loss of any single drive. 
+At the highest redundancy levels, MinIO can serve read requests with minimal performance impact despite the loss of up to half (:math:`N / 2`) of the total drives in the deployment.
+
+MinIO calculates the size and number of Erasure Sets in a Server Pool based on the total number of drives in the set *and* the number of :mc:`minio` servers in the set. See :ref:`minio-ec-erasure-set` for more information.
+
+Correct errors automatically with healing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Healing is MinIO's ability to restore data after some event causes data loss.
+Data loss can come from bit rot, drive loss, or node loss.
+
+:ref:`Erasure coding <minio-erasure-coding>` provides continued read and write access if an object has been partially lost.
+
+Depending on configurations, MinIO can tolerate losing up to half of the available drives on a deployment without losing data.
+
+Write data protection at the object level with parity
```

Comment:
I'd just merge this up into the Erasure Coding section, and simplify as much as possible while pointing to the EC docs.